Build/Common.Build.settings

```diff
@@ -144,4 +144,38 @@
   <PropertyGroup>
     <CodeAnalysisRuleSet>$(BuildRoot)\Build\xTVS.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <MajorVersion>1</MajorVersion>
+    <MinorVersion>4</MinorVersion>
+
+    <!-- This was reset for Dev15. It needs to eventually be reset to the current year for Dev16 -->
```

Comment:
I still like to reset the year every so often to make sure the build numbers < int16.max